scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -146,8 +146,9 @@ <h3 class="subnav__page-title">
                             <i class="icon-switches"></i>
                             &nbsp;<span>{{:: 'Rearrange Groups' | translate}}</span>
                         </button>
-                        <button ng-if="swimlane" ng-click="switchSwimlaneColumn()">{{ :: 'Show' | translate }} <span ng-show="!showColumns6" class="ng-hide">6</span>
-                        <span ng-if="showColumns6" class="ng-scope">5</span>{{:: ' columns' | translate}}</button>
+                    </li>
+                    <li ng-if="swimlane && type === 'monitoring'" ng-repeat="n in [2,3,4,5,6]">
```

Comment:
Is there a reason you restricted it to monitoring view?

Reply:
If `swimlane` view is only used in monitoring, the condition `ng-if="swimlane"` already covers this. Remove the `type === 'monitoring'` check since it's redundant and a bit confusing by implying that there is something page-specific going on when in fact there isn't.